forem/forem
# Add more visibility and control into the Admin Article page

## What type of PR is this? (check all applicable)


## Description

This PR adds more details to the `Admin Article Page` so that admins can have more info to make any decision.

## Related Tickets & Documents



- Related Issue #19314

## QA Instructions, Screenshots, Recordings

**How to test?**
1. Sign in as admin locally
2. Visit http://localhost:3000/admin/content_manager/articles or any specific article like http://localhost:3000/admin/content_manager/articles/1 or http://localhost:3000/admin/content_manager/articles/2 ,etc
3. All these pages should look almost same and focus that right next to the title of article we show `thumbs-up`, `thumbs-down`, `vomit` and `score`. 
4. You can edit these information by going to any specific article, click on `Mod` icon available on left hand side of the article-view
5. This will open the menu from right 
6. In this menu you can mark it as high-quality, low-quality or you can flag it.

Loom Output: https://www.loom.com/share/1b665f146b264d2eaf6ac7fdeb1e7206

| Before  | After |
| ------------- | ------------- |
| <img width="997" alt="Screenshot 2023-04-18 at 11 19 18 PM" src="https://user-images.githubusercontent.com/9396084/232861792-92095aa4-f14f-4e10-a949-3ccc266e199e.png"> | <img width="1008" alt="Screenshot 2023-04-18 at 11 18 27 PM" src="https://user-images.githubusercontent.com/9396084/232861749-a29fc0f1-8af2-4939-b2db-990be5b47b89.png"> |

### UI accessibility concerns?

No

## Added/updated tests?
